Formula/hbase.rb

```diff
@@ -1,9 +1,8 @@
 class Hbase < Formula
   desc "Hadoop database: a distributed, scalable, big data store"
   homepage "https://hbase.apache.org"
-  url "https://www.apache.org/dyn/closer.cgi?path=hbase/1.1.5/hbase-1.1.5-bin.tar.gz"
-  sha256 "e06fa399c7ba48acd14b71ba6316d272f14b18b5b7398d6a3d60b6b8a9073f0e"
-  revision 1
```

Comment:
The audit was correct 👍. Formula revisions are reset to 0 every stable version update, and 0 is unwritten so the line just gets chopped.